src/App/Patterns/content/Patterns/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { DocContainer } from "@docutils";
+import { Link } from "react-router-dom";
+import patternList from "@src/App/routes/patterns";
```

Comment:
*nitpick* we're importing routes here, for consistency with other files that import routes:
patternRoute/patternRoutes?